QEfficient/base/onnx_transforms.py

```diff
@@ -53,9 +53,32 @@ def apply(cls, model: ModelProto, onnx_base_dir: Optional[str] = None) -> Tuple[
         return model, transformed
 
 
-class SplitWeights(OnnxTransform):
-    pass
-
+class SplitTensorsTransform(OnnxTransform):
+    """
+    Split external tensors file
+    """
 
-class LoraAdapters(OnnxTransform):
-    pass
+    @classmethod
+    def apply(
+        cls,
+        model: ModelProto,
+        *,
+        model_name: str,
+        onnx_base_dir: Optional[str] = None,
+        file_chunk_size: int = 10 * 2**30,  # 10 GiB
+        size_threshold: int = 1024,
```

Comment:
What is the purpose of `size_threshold`?

Reply:
Only tensors greater than this threshold (in bytes) will be saved externally.

Added docstring to clarify the same.